src/PresentationalComponents/ZeroState/AppZeroState.test.js

```diff
@@ -14,7 +14,14 @@ jest.mock('@redhat-cloud-services/frontend-components-utilities/interceptors', (
     }))
 }));
 
-const appNames = Object.keys(zeroStateAppList).map(key => key.split('_')[0]);
+const appNames = Object.keys(zeroStateAppList).map(key => {
+    let app = key.split('_')[0];
+    if (app === 'CONTENT' || app === 'RESOURCE') {
+        app = `${app}_${key.split('_')[1]}`;
+    }
```

Comment:
I think we can remove `if` condition and just have something like `let app = key.replace('_ZERO_STATE', '');`

Reply:
Love this fix!!!